tests/src/TestFixtureBase.cpp

```diff
@@ -108,11 +106,6 @@ void TestFixtureBase::LoadIds()
 {
 	if (m_serverUrl.length() > 0) return; //check if loaded
 
-	//Get current working directory
```

Comment:
This current dir is not used anywhere in this code.
The `getcwd` was causing issue on some platforms, so that's a good thing 👍